wireguard/libwg/build-android.sh

```diff
@@ -46,19 +46,23 @@ for arch in ${ARCHITECTURES:-armv7 aarch64 x86_64 i686}; do
     echo $(pwd)
     make -f Android.mk clean
 
-    export CFLAGS="-D__ANDROID_API__=21"
+#    this is determined by the NDK
+#    export CFLAGS="-D__ANDROID_API__=21"
 
     make -f Android.mk
 
     # Strip and copy the libray to `android/build/extraJni/$ANDROID_ABI` to be able to build the APK
     UNSTRIPPED_LIB_PATH="../../build/lib/$RUST_TARGET_TRIPLE/libwg.so"
-    STRIPPED_LIB_PATH="../../android/app/build/extraJni/$ANDROID_ABI/libwg.so"
+    STRIPPED_LIB_PATH="../../build/lib/extraJni/$ANDROID_ABI/libwg.so"
```

Comment:
To minimise the changes in the `wireguard` directory, as this is somewhat a fork of upstream, could we not keep this path is is and just adapt the `.cargo/config.toml` paths?